app/models/media/image.rb

```diff
@@ -0,0 +1,8 @@
+class Media::Image < Media::Object
+  has_attached_file :item, styles: {
+    normal: '720x720>',
+    thumb: '100x100#'
+  }
```

Comment:
No issues, as far as I can tell. Under the hood, this is just using `define_method` for a bunch of different methods, so the child overrides the parent. It successfully defines and generates the files for those attachments when they're created.